pending-documentation.md

```diff
@@ -459,6 +459,20 @@ each argument of the production, whereas `strict(c; 1)` will instantiate it
 only for the first argument. The special variable `HERE` is used to tell the
 compiler where you want to place the production that is to be heated or cooled.
 
+You can also specify multiple context aliases for different parts of a production,
+for example:
+
+```k
+syntax Exp ::= foo(Exp, Exp) [strict(left; 1; right; 2)]
```

Comment:
```
syntax Exp ::= foo(Exp, Exp) [strict(1(left); 2(right)]
```
Wouldn't this be more intuitive? In this case, you might need to add a new parsing entry in `Outer.jj`, but it shouldn't be that hard.

Reply:
I agree with @dwightguth that it's inconsistent, in other places where we need lists or 2D lists of data we always use `_;_` and `_;_,_,_;_`. If we want improved ways of specifying structured data in attributes, that can be another PR.